thunderhub.patch

```diff
@@ -1,6 +1,6 @@
---- thunderhub/arm64v8.Dockerfile	2022-04-20 12:52:38.319638554 -0600
+--- thunderhub/Dockerfile	2022-04-20 12:52:38.319638554 -0600
```

Comment:
I'm mildly surprised this patch file works. Did you generate it via a diff?

Reply:
nope. I manually edited it after reverse engineering yours.

Reply:
if you attempt to do a build that applies this patch, does it behave as expected?

Reply:
the health check timing seems to be off, but that's a different bug. 
![Screenshot from 2022-06-27 12-42-42](https://user-images.githubusercontent.com/34528298/175992488-a14666fc-bb8e-4fa4-87fd-b8a3c28d5b8d.png)
![Screenshot from 2022-06-27 12-42-34](https://user-images.githubusercontent.com/34528298/175992512-a660eda6-61a6-49d7-9f34-c6d6811eb450.png)